### PR TITLE
chore: update scaffolding to new tools and settings

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
not at the moment!

**Comment by itajaja** [comment]
how does it figure out eg if you need to do `tsc --noEmit`? by looking if there is a tsconfig?

File: packages/init/templates/package.json.hbs [hunk 1]
```diff
@@ -100,7 +76,9 @@
   {{/unless}}
 
   "devDependencies": {
+    "@4c/cli": "^0.7.12",
     "@4c/jest-preset": "^1.3.0",
+    "@4c/import-sort": "^4.3.3",
     {{#if babel}}
       "@babel/cli": "^7.2.3",
       "@babel/core": "^7.2.2",
```
Comment (taion) [inline packages/init/templates/package.json.hbs]: remove the babel-core bridge below

**Comment by jquense** [comment]
> by looking if there is a tsconfig? 

yeah 👍

**Pull request merged**